pkg/repository/cachedep.go

```diff
@@ -169,7 +169,7 @@ func (i *invalidate) Exec(newTx bool) error {
 	if newTx { //start new DB transaction
 		return i.transactional("invalidating cache entries", dbOps)
 	}
-	return dbOps() //no new DB transaction requested
+	return dbOps(nil) //no new DB transaction requested
```

Comment:
as above... Exec has to us a provided `db.Connection` reference and not a boolean

Reply:
Fixed